data-raw/396_fix-xstr-outputs.Rmd

```diff
@@ -0,0 +1,165 @@
+---
+output: github_document
+params:
+  import_product_level: "~/Downloads/pstr_product_level_v2.csv"
+  import_company_level: "~/Downloads/pstr_company_level_v2.csv"
+  export_product_level: "~/Downloads/pstr_product_level_v2_fixed.csv"
+  export_company_level: "~/Downloads/pstr_company_level_v2_fixed.csv"
```

Comment:
@kalashsinghal and @Tilmon  you can change this parameters to import and export different files.